crates/ty_python_semantic/resources/mdtest/typed_dict.md

```diff
@@ -190,20 +202,30 @@ These attributes can not be accessed on inhabitants:
 
 ```py
 def _(person: Person) -> None:
-    # TODO: these should be errors
-    person.__total__
-    person.__required_keys__
-    person.__optional_keys__
+    person.__total__  # error: [unresolved-attribute]
+    person.__required_keys__  # error: [unresolved-attribute]
+    person.__optional_keys__  # error: [unresolved-attribute]
 ```
 
 Also, they can not be accessed on `type(person)`, as that would be `dict` at runtime:
 
 ```py
-def _(t_person: type[Person]) -> None:
-    # TODO: these should be errors
-    t_person.__total__
-    t_person.__required_keys__
-    t_person.__optional_keys__
+def _(person: Person) -> None:
+    type(person).__total__  # error: [unresolved-attribute]
+    type(person).__required_keys__  # error: [unresolved-attribute]
+    type(person).__optional_keys__  # error: [unresolved-attribute]
+```
+
+But they *can* be accessed on `type[Person]`, because this function would accept `Person` as an
+argument:
```

Comment:
what you have is accurate as-is, but this might be clearer for a first-time reader:

```suggestion
But they *can* be accessed on `type[Person]`, because this function would accept the class object
`Person` as an argument:
```